ice-hockey-manager-service/src/test/java/cz/muni/fi/pa165/facade/TeamFacadeTest.java

```diff
@@ -0,0 +1,105 @@
+package cz.muni.fi.pa165.facade;
+
+import cz.muni.fi.pa165.config.PersistenceConfiguration;
+import cz.muni.fi.pa165.dto.TeamCreateDTO;
+import cz.muni.fi.pa165.dto.TeamDTO;
+import cz.muni.fi.pa165.enums.CompetitionCountry;
+import cz.muni.fi.pa165.service.config.ServiceConfiguration;
+import cz.muni.fi.pa165.service.facade.TeamFacade;
+import cz.muni.fi.pa165.service.facade.TeamFacadeImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestExecutionListeners;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.springframework.test.context.transaction.TransactionalTestExecutionListener;
+import org.springframework.transaction.annotation.Transactional;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import static org.assertj.core.api.Assertions.*;
```

Comment:
It is a good practice not to use .* when importing classes.

Reply:
ok